cmake/InstallArangoDBJSClient.cmake

```diff
@@ -29,11 +29,24 @@ if (USE_ENTERPRISE)
   )
 endif ()
 
-# For the node modules we need all files:
+# For the node modules we need almost all files:
 install(
   DIRECTORY ${ARANGODB_SOURCE_DIR}/js/node
   DESTINATION ${CMAKE_INSTALL_DATAROOTDIR_ARANGO}/${ARANGODB_JS_VERSION}
   REGEX "^.*/eslint"                                       EXCLUDE
   REGEX "^.*/.npmignore"                                   EXCLUDE
   REGEX "^.*/.bin"                                         EXCLUDE
+  REGEX "^.*/package.*json"                                EXCLUDE
```

Comment:
I think this line was too much, as it breaks all node_modules in an installed version.
```suggestion
```